script.js

```diff
@@ -0,0 +1,354 @@
+    {className:"ArrowRight",eng:{caseDown:"►",caseUp:"►"},rus:{caseDown:"►",caseUp:"►"}},
+    {className:"ControlRight",eng:{caseDown:"Ctrl",caseUp:"Ctrl"},rus:{caseDown:"Ctrl",caseUp:"Ctrl"}}];
+
+
+const specialKey=["Backspace","Tab","Enter","CapsLock","ShiftLeft","ShiftRight","AltLeft","AltRight","ControlLeft","ControlRight","MetaLeft","Delete", "Space"];
+const engAlphabet=[
+    "KeyQ", "KeyW", "KeyE", "KeyR", "KeyT", "KeyY", "KeyU", "KeyI", "KeyO", "KeyP", 
+    "KeyA", "KeyS", "KeyD","KeyF","KeyG","KeyH","KeyJ","KeyK","KeyL",
+    "KeyZ", "KeyX", "KeyC", "KeyV", "KeyB", "KeyN", "KeyM",
+];
+
+const rusAlphabet=[
+    "Backquote", 
+    "KeyQ", "KeyW", "KeyE", "KeyR", "KeyT", "KeyY", "KeyU", "KeyI", "KeyO", "KeyP", "BracketLeft", "BracketRight",
+    "KeyA", "KeyS", "KeyD","KeyF","KeyG","KeyH","KeyJ","KeyK","KeyL", "Semicolon", "Quote",
+    "KeyZ", "KeyX", "KeyC", "KeyV", "KeyB", "KeyN", "KeyM", "Comma", "Period",
+];
+
+// STATE
+
+let state = {
+    lang: localStorage.getItem('lang') || 'eng',
+    caps: false,
+    shift: false,
+}
+
+window.addEventListener('unload', () => {
+    localStorage.setItem('lang', state.lang)
+})
+
+
+const BODY =  document.body;
+const WRAPPER = document.createElement('div');
+
+WRAPPER.setAttribute('class', 'wrapper');
+WRAPPER.innerHTML = `
+        <textarea name="text" id="text" class="textarea-keyboard" cols="100" rows="10" autofocus></textarea>
+        <div class="keyboard" id="keyboard"></div>
+        <div class="instructions">
+            <p> Клавиатура создана в операционной системе Linux Mint 19.3 Tricia </p>
+            <p> Для переключения языка комбинация: левыe ctrl + alt </p>
+        </div>
+
+`
+BODY.append(WRAPPER);
+
+const KEYBOARD = document.querySelector('#keyboard');
+const TEXTAREA = document.querySelector('textarea');
+
+// Draw Keyboard
+
+const drawKey = (key, type, lang) => {
+    return `
+        <button class="${type} ${key.className} key">
+            ${key[lang].caseDown}
+        </button>
+   `;
+}
+
+const drawSpecialKey = (key) => {
+    if(["Backspace","Enter","CapsLock","ShiftLeft", "ShiftRight"].includes(key.className)) {
+        KEYBOARD.insertAdjacentHTML("beforeend", drawKey(key, 'key__special-double', state.lang));
+    } else {
+        if(key.className === 'Space'){
+            KEYBOARD.insertAdjacentHTML("beforeend", drawKey(key, 'key__special-space', state.lang));
+        } else {
+            KEYBOARD.insertAdjacentHTML("beforeend", drawKey(key, 'key__special-simple', state.lang));
+        }
+    }
+    
+}
+
+const drawKeyboard = (keys) => {
+    for(let row of keys){
+        for(let key of row){
+            if(specialKey.includes(key.className)){
+                drawSpecialKey(key);
+            } else {
+                KEYBOARD.insertAdjacentHTML("beforeend", drawKey(key, 'key__simple', state.lang));
+            }
+            
+        }
+    }
+} 
+
+drawKeyboard(a);
+
+// render keys
+
+function renderKeys (key) {
```

Comment:
Also `renderKeys` and `drawKeys` functions naming looks so similar but does the different things. It's a confusing naming that impact to the code readability.

I can suggest this:
```suggestion
function updateKeyboardLayout (key) {
```